pageScraper.js

```diff
@@ -1,91 +1,48 @@
-// const isbns = ["4065294266", '4022650591', '9781599664026']
-const isbns = ['9781599664026']
-const urls = {
-  "Booklog": ["https://booklog.jp/item/1/", ".item-info-author"],
-  "Bookfinder": [
-    "https://www.bookfinder.com/search/?isbn=",
-    "&mode=isbn&st=sr&ac=qr",
-    ".attributes"
-  ],
-}
-// todo: 
-//  - make "urls" an object to make it more managable
-//  - use args to customize puppeteer queries (page.$eval()) -> customization depending on the sites
-async function bookfinderQuery(isbn, browser) {
-  // Function that queries bookLog site
-  let url1 = urls["Bookfinder"][0];
-  let url2 = urls["Bookfinder"][1];
-  let waitSelector = urls["Bookfinder"][2];
-  return await pagePromise(url1, url2, waitSelector, isbn, browser)
-}
-
-async function booklogQuery(isbn, browser) {
-  // Function that queries bookLog site
-  let url1 = urls["Booklog"][0];
-  let url2 = null;
-  // url1 = first half of the url
-  // url2 = second half of the url
-  let waitSelector = urls["Booklog"][1];
-  return await pagePromise(url1, url2, waitSelector, isbn, browser)
-}
-
```

Comment:
Extracting all of the site specific details to the urls config took away the need for specialized code for each site.